notebooks/254-llm-chatbot/254-llm-chatbot.ipynb

```diff
@@ -116,7 +116,7 @@
     {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        num_attention_heads = (ov_model.normalized_config.num_attention_heads if ov_model.config.model_type == "bloom" else 1)
</pre><p>probably this condition can be simplified as there is no bloom support in notebook and you calibrate only red-pajama that 100% is not bloom</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1382/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>thanks for noticing this. Simplified, now looks better</p>